lib/Sema/TypeCheckGeneric.cpp

```diff
@@ -24,6 +24,7 @@
 #include "swift/AST/TypeCheckRequests.h"
 #include "swift/AST/TypeResolutionStage.h"
 #include "swift/AST/Types.h"
+#include "swift/AST/ASTMangler.h"
```

Comment:
The two changes in this file no longer appear to be necessary, can you squash and remove them maybe?